src/data/columns.js

```diff
@@ -0,0 +1,32 @@
+export const columns = [
+  {
+    title: 'Asylum Office',
+    dataIndex: 'asylumOffice',
```

Comment:
I don't feel like anyone will have to look up the naming convention. Way to keep it simple